packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -9,10 +9,11 @@ export interface CheckboxProps extends Omit<HTMLProps<HTMLInputElement>, 'value'
   label?: string;
   description?: string;
   value?: boolean;
+  size?: number;
```

Comment:
If we are to add something like this it would use ComponentSize (sm, md, lg) not a number 

but since we have not use case for this yet not sure we want to add this support